src/components/POLine/Item/ItemView.js

```diff
@@ -1,43 +1,56 @@
-import React from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
+import get from 'lodash/get';
+import toString from 'lodash/toString';
 import { KeyValue, Row, Col } from '@folio/stripes/components';
+import ProductIdDetails from './ProductIdDetails';
 
-class ItemView extends React.Component {
+class ItemView extends Component {
   static propTypes = {
-    initialValues: PropTypes.object
+    itemDetails: PropTypes.object
```

Comment:
Please, update to `PropTypes.object.isRequired` if it is a required prop or add `static defaultProps` field otherwise